File: Review/20211003/printNumbers.go
package _0211003

func printNumbers(n int) []int {
	res:=[]int{}
	ans:=0
	for n!=0{
		ans=ans*10+9
		n--
	}
	for i:=1;i<=ans;i++{
		res=append(res,i)
	}
	return res
}

File: Leetcode/剑指offer/easy/20210913/printNumbers.cpp
class Solution {
public:
    vector<int> printNumbers(int n) {
        int res=0;
        vector<int>ans;
        for(int i=1;i<=n;i++){
            res=res*10+9;
        }
        for(int i=1;i<=res;i++){
            ans.push_back(i);
        }
        return ans;
    }
};
